tests/resources/swp_categories.json

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
As of now, the tests are still failing. The responses are already there - do you plan to fix the tests?

Reply:
Okay.